Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -150,14 +148,16 @@ void ABenchmarkGymGameModeBase::TryInitialiseExpectedActorCounts()
 
 void ABenchmarkGymGameModeBase::BuildExpectedActorCounts()
 {
-	AddExpectedActorCount(ExpectedNPCsCount, NPCClass, TotalNPCs, 1);
-	AddExpectedActorCount(ExpectedSimPlayersCount, SimulatedPawnClass, ExpectedPlayers, 1);
+	// Zoning scenarios can report actor count numbers slightly higher than the expected number so add a little slack.
```

Comment:
Might not be true anymore

Reply:
Does this comment need updating?

Reply:
They still have the potential to go over but from what I have seen the numbers vary a lot less (I have only ever seen the numbers go over by 1)